### ISSUE TITLE
Search box clips text when resized

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6771]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `search for a user` box on the `Manage Proxies` page clips text when resized and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/proxies?locale=en and run the Site Improve browser extension

![text_clip](https://github.com/samvera/hyrax/assets/32644254/cb2270a4-3ade-4227-8885-5255494dfd4f)

**Comment by christhepianist** [comment]
Tested and confirmed that this is no longer an issue.
